### PR TITLE
Protocols new properties described

### PR DESCRIPTION
**Pull request by semuserable**

### CODE REVIEW

**Comment by semuserable** [comment]
Please, review and run authorized deploy for Vercel. Can't add reviewers, btw..

cc @bruno-garcia

File: src/docs/sdk/event-payloads/contexts.mdx [hunk 1]
```diff
@@ -149,6 +149,54 @@ The `type` and default key is `"device"`.
...
+`cpu_description`
+
+: _Optional_. CPU description. For example, `Intel(R) Core(TM)2 Quad CPU Q6600 @ 2.40GHz`.
+
+`processor_frequency`
+
+: _Optional_. Processor frequency in MHz. Note that the actual CPU frequency might vary depending on current load and power conditions, 
+  especially on low-powered devices like phones and laptops. On some platforms it's not possible to query the CPU frequency. 
+  Currently such platforms are iOS and WebGL. For example, `2500`.
+
```
Comment (bruno-garcia) [inline src/docs/sdk/event-payloads/contexts.mdx:165]:
```suggestion
  especially on low-powered devices like phones and laptops.

```

File: src/docs/sdk/event-payloads/contexts.mdx [hunk 2]
```diff
@@ -149,6 +149,54 @@ The `type` and default key is `"device"`.
...
+`battery_status`
+
+: _Optional_. Status of the device's battery. For example, `Unknown, Charging, Discharging, NotCharging, Full`.
+
+`device_unique_identifier`
+
+: _Optional_. Unique device identifier. Depends on the running platform. For example, `iOS: UIDevice.identifierForVendor (UUID)`,
+  `Android: md5 of ANDROID_ID`, `Windows Store Apps: AdvertisingManager::AdvertisingId (possible fallback to HardwareIdentification::GetPackageSpecificToken().Id)`,
+  `Windows Standalone: hash from the concatenation of strings taken from Computer System Hardware Classes`.
```
Comment (bruno-garcia) [inline src/docs/sdk/event-payloads/contexts.mdx:178]:
```suggestion
: _Optional_. Unique device identifier. This value might only be used if `sendDefaultPii` is enabled.
```

**Review by untitaker — APPROVED** [review]
I'd ask you to add them to https://github.com/getsentry/relay/blob/master/relay-general/src/protocol/contexts.rs as well.

File: src/docs/sdk/event-payloads/contexts.mdx [hunk 3]
```diff
@@ -149,6 +149,54 @@ The `type` and default key is `"device"`.
...
 : _Optional_. The language of the device. For example, `en-US`.
 
+`processor_count`
+
+: _Optional_. Number of "logical processors". For example, `8`.
+
+`cpu_description`
+
+: _Optional_. CPU description. For example, `Intel(R) Core(TM)2 Quad CPU Q6600 @ 2.40GHz`.
+
```
Comment (untitaker) [inline src/docs/sdk/event-payloads/contexts.mdx:159]: I'm just curious, why do you need this when you already report the phone model?

Comment (bruno-garcia) [reply src/docs/sdk/event-payloads/contexts.mdx:159]: I personally found that interesting. Possibly even more to those building games that run on a variety of devices like desktop, console and mobile.

**Comment by bruno-garcia** [comment]
> I'd ask you to add them to https://github.com/getsentry/relay/blob/master/relay-general/src/protocol/contexts.rs as well.

Keen to write some Rust? Probably don't even need to compile it locally :D

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**